eventbridge-bedrock-s3-aoss/README.md

```diff
@@ -0,0 +1,196 @@
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd serverless-patterns/eventbridge-bedrock-s3-aoss
+    ```
+1. Create virtual environment for Python
+    ```
+    python3 -m venv .venv
+    ```
+1.  Activate the virtualenv like this:
+    
+     ```
+    source .venv/bin/activate
+    ```
+1. Install the Python required dependencies:
+    ```
+    pip install -r requirements.txt
+    ```
+1. Install dependencies to be used in Lambda Layer
+    
+    ```
+    pip install --target layers/python -r layers/requirements.txt
+    ```
+
+1. Run the command below to bootstrap your account. CDK needs it to deploy
+    ```
+    cdk bootstrap
+    ```
+1. see the list of the IDs of the stacks in the AWS CDK application:
+    ```
+    cdk list
+    ```
+
+1. Review the CloudFormation template CDK generates for the included stacks using the following AWS CDK CLI command:
+
+
+> [!NOTE]  
+> Substitute the stack_id with one from the list in output from the `cdk list` command
+    ```
+    cdk synth <stack_id>
+    ```
+
+10. From the command line, use AWS CDK to deploy the AWS resources.
+
+    ```
+    cdk deploy --all
+    ```
+    Enter `y` if prompted `Do you wish to deploy these changes (y/n)?`
+    
+> [!NOTE]  
+> You can optionally change the `collection_name`, `index_name`, `knowledge_base_name`, `kb_s3_datasource_name`
+parameters in the `cdk.context.json`. The parameters are used to name the OpenSearch Serverless collection, index, the Amazon Bedrock Knowledge base and the associated S3 data source, respectively.
+
+## How it works
+Upon deployment, the CDK stack will create a Knowledge Base for Bedrock configure with S3 Bucket as data source and an OpenSearch Serverless collection to store vector data. A data source repository contains files or content with information that can be retrieved when your knowledge base is queried. The stack also include an EventBridge scheduler that is configured to run every 5 mins and invoke the `StartIngestionJob` API on Amazon Bedrock Agents service. Amazon Bedrock supports a monitoring system to help you understand the execution of any data ingestion jobs. The Stack would create the neccessary CloudWatch log groups and CloudWatch delivery. You can gain visibility into the ingestion of your knowledge base resources with this logging system. Additionally, Amazon Bedrock is integrated with AWS CloudTrail, a service that provides a record of actions taken by a user, role, or an AWS service in Amazon Bedrock. CloudTrail captures all API calls for Amazon Bedrock as events.
```

Comment:
Upon deployment, the CDK stack will create a Knowledge Base for Bedrock configured with S3 Bucket as data source and an OpenSearch Serverless collection to store vector data.

Reply:
Pls use Amazon Bedrock at the beginning and for rest of the file use Bedrock. Same naming standard for all other AWS services.

Reply:
updated service names accordingly and fixed grammar